src/main/java/com/softserve/teamproject/entity/Group.java

```diff
@@ -162,17 +175,17 @@ public int hashCode() {
     return Objects.hash(name);
   }
 
-  @Override
-  public String toString() {
-    return "Group{"
-        + "id=" + id
-        + ", name='" + name + '\''
-        + ", teachers=" + teachers
-        + ", location=" + location
-        + ", startDate=" + startDate
-        + ", finishDate=" + finishDate
-        + ", status=" + status
-        + ", specialization=" + specialization
-        + '}';
-  }
+  //  @Override
+  //  public String toString() {
+  //    return "Group{"
+  //        + "id=" + id
+  //        + ", name='" + name + '\''
+  //        + ", teachers=" + teachers
+  //        + ", location=" + location
+  //        + ", startDate=" + startDate
+  //        + ", finishDate=" + finishDate
+  //        + ", status=" + status
+  //        + ", specialization=" + specialization
+  //        + '}';
+  //  }
 }
```

Comment:
It doesn't seem to be the best approach of solving issues like that. I wouldn't recommend merging that in master